lib/data_pipelines/customer_io_method_channel.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+
+import '../customer_io_config.dart';
+import '../customer_io_enums.dart';
+import '../extensions/method_channel_extensions.dart';
+import '_native_constants.dart';
+import 'customer_io_platform_interface.dart';
+
+/// An implementation of [CustomerIOPlatform] that uses method channels.
+class CustomerIOMethodChannel extends CustomerIOPlatform {
```

Comment:
Updated implementation of `lib/customer_io_method_channel.dart` using extensions